supabase/client.ts

```diff
@@ -0,0 +1,15 @@
+import { createClient } from '@supabase/supabase-js';
+import { Database } from './types';
+
+const development = true;
+export const JWT_SECRET = 'ncjn3kqBv9eODXXpKYW9GbvZcT6gswzg9w+6/Jbty+DcQvxz9ZUDr0UpmBet6tFV4CwAVB7lmNiI71r5JC1h7g==';
```

Comment:
I guess all of these secrets should be added as Cloudflare secrets? If we aim to open source this repository, we cannot have any secrets in it code.

Reply:
Yes, I left that for the end and forgot to clean it up 🤦‍♀️

Reply:
Since the repository is already public, I suggest you roll/reset all the exposed secrets from this PR 🤷‍♂️ I believe you can do that from Supabase's UI?

Reply:
Yes, you are right. I will do that 😉